package.json

```diff
@@ -24,10 +24,8 @@
   },
   "homepage": "https://github.com/kiwicom/vault2env-js#readme",
   "dependencies": {
-    "co": "^4.6.0",
     "fs-extra": "^4.0.1",
     "minimist": "^1.2.0",
-    "request": "^2.81.0",
```

Comment:
request is peer dependency for `request-promise-native`

`request is defined as a peer-dependency and thus has to be installed separately.`

Reply:
Omg, why is `yarn` silent about this? Fixed...